packages/kernel-6.1/kernel-6.1.spec

```diff
@@ -66,7 +66,7 @@ Requires: %{name}-modules = %{version}-%{release}
 Requires: %{name}-devel = %{version}-%{release}
 
 # Pull in default mkfs conf for xfsprogs.
-Requires: (%{name}(mkfs-xfs-conf) if %{_cross_os}xfsprogs)
+Requires: (%{name}-mkfs-xfs-conf if %{_cross_os}xfsprogs)
```

Comment:
Also this line, shouldn't  be in this commit, you have a commit for the 6.1 kernel